### DISCUSSION TITLE
Exposure Time Finder, adjust ALL text size

### DISCUSSION THREAD

**Discussion by bryanseger** [sn4k3/UVtools#955] [Ideas]
Adjusting the text scale & thickness in the Exposure Time Finder only adjusts the 'text' field of the exposure strips (in green below), but not the label on the lower-left of the strip (in yellow below). It would be nice if this label text can be adjusted too, especially as resolutions get finer and finer. These tiny letters can sometimes get "lost" in a vat if they detach, so thickening them would be helpful.

![image](https://github.com/user-attachments/assets/05b8b2ef-6708-4eda-8d05-bced47adbe5a)

**Comment by sn4k3** [comment]
This was requested multiple times now, I will try to make it work.
But one way to do it, is to half all your x/y sizes of features and then you use the tool resize to double everything, you will get everything in correct size plus the text doubled
